assets/thumbnails/model_list.yaml

```diff
@@ -128,3 +128,6 @@ models:
 - OpenRobotics/WoodenChair
 - OpenRobotics/XRayMachine
 - OpenRobotics/XRayMachineSmall
+actor_models:
+- OpenRobotics/actor
```

Comment:
There is no `actor` model under OpenRobotics, there is one under Mingfei but it doesn't really work because of the different coordinates that it uses, screenshot below:

![image](https://user-images.githubusercontent.com/9111558/104431147-88ad3080-55c2-11eb-87da-076e7b1c2a45.png)

I would suggest removing it